src/app/core/ui/navigation/menu-item/menu-item.component.html

```diff
@@ -22,10 +22,4 @@
   </a>
 </mat-list-item>
 
-<!-- Render submenus if they exist and are expanded -->
-<div *ngIf="isExpanded" class="submenu">
-  <ng-container *ngFor="let subItem of item.subMenu">
-    <!-- Pass activeLink to each submenu item to highlight the active submenu -->
-    <app-menu-item [item]="subItem" [activeLink]="activeLink"></app-menu-item>
-  </ng-container>
-</div>
+
```

Comment:
I think now you removed the previous, working logic to display (not edit) submenu items here?